src/SOS/Strike/platform/targetimpl.cpp

```diff
@@ -96,36 +96,32 @@ ITarget* Target::GetInstance()
 HRESULT 
 Target::CreateInstance(IRuntime **ppRuntime)
 {
-    HRESULT hr = S_OK;
-    if (*ppRuntime == nullptr)
-    {
-        hr = Runtime::CreateInstance(this, IRuntime::Core, &m_netcore);
+    HRESULT hr = Runtime::CreateInstance(this, IRuntime::Core, &m_netcore);
 #ifdef FEATURE_PAL
-        * ppRuntime = m_netcore;
+    *ppRuntime = m_netcore;
 #else
-        ITarget::OperatingSystem os = this->GetOperatingSystem();
-        switch (os)
-        {
-            case ITarget::OperatingSystem::Linux:
-            case ITarget::OperatingSystem::OSX:
-                // Only attempt to create linux/OSX core runtime if above failed
-                if (m_netcore == nullptr)
-                {
-                    hr = Runtime::CreateInstance(this, IRuntime::UnixCore, &m_netcore);
-                }
-                break;
-            case ITarget::OperatingSystem::Windows:
-                // Always attempt to create desktop clr, but only return result the if the .NET Core create failed
-                HRESULT hrDesktop = Runtime::CreateInstance(this, IRuntime::WindowsDesktop, &m_desktop);
-                if (m_netcore == nullptr)
-                {
-                    hr = hrDesktop;
-                }
-                break;
-        }
-        *ppRuntime = m_netcore != nullptr ? m_netcore : m_desktop;
-#endif
+    ITarget::OperatingSystem os = this->GetOperatingSystem();
+    switch (os)
+    {
+        case ITarget::OperatingSystem::Linux:
+        case ITarget::OperatingSystem::OSX:
+            // Only attempt to create linux/OSX core runtime if above failed
+            if (m_netcore == nullptr)
+            {
+                hr = Runtime::CreateInstance(this, IRuntime::UnixCore, &m_netcore);
+            }
+            break;
+        case ITarget::OperatingSystem::Windows:
+            // Always attempt to create desktop clr, but only return result the if the .NET Core create failed
```

Comment:
Why is this different than the linux/OSX case? (always attempt to create desktop clr vs only attempt to create linux/OSX core runtime if above failed). 

Does desktop core refer to .NET framework?

Reply:
"desktop" means .NET Framework.  This code under Windows can create both a .NET Core (m_netcore) and a .NET Framework (m_desktop) runtime. A target process can have both.

This code is conditionally built for the Windows version of SOS but can target Linux/OSX which don't have .NET Framework runtimes.